CoreGraphFramework/src/au/gov/asd/tac/constellation/graph/schema/type/SchemaTransactionType.java

```diff
@@ -268,15 +268,28 @@ public Builder setIncomplete(final boolean incomplete) {
         }
 
         public SchemaTransactionType build() {
+            ConstellationColor newColor = color;
+            LineStyle newStyle = style;
+            boolean newDirected = false;
```

Comment:
I think you'll need to set up `newDirected` similar to the other 2. Simply because in the case that `directed` is true, you'll pass the wrong value in the return. Consequently you will need to make `newDirected` the Boolean object since `directed` could be null